### DISCUSSION TITLE
Enforcing a TAU request from UE

### DISCUSSION THREAD

**Discussion by 3rt1m** [srsran/srsRAN_4G#1156] [General]
Hello,

I am relatively new to LTE technology and I am trying to simulate a TAU request on a UE and wanted to know if this would be a way to achieve this.
Scenario:
Have 2 different srsran base stations (eNB-1 and eNB-2) within a RF screen room with different tac values each.
Both eNB-1 and eNB-2 have same earcfn and PLMN values to allow the UE to connect to them.
Have the UE device connect to eNB-1 and shortly after start the eNB-2.
Then turn off the eNB-1 to force the UE to do a handover to eNB-2.
I assume that the tac values being different will force the UE to perform a TAU request in this case.

Please let me know if this would work and if not what are some better options based on my setup?

**Comment by andrepuschmann** [comment]
Hey, I think that could work. However, instead of a handover the UE would probably to a reestablishment into the new cell. Since you're killing the cell (I assume) there won't be any real handover happening.
